README.md

```diff
@@ -34,32 +39,52 @@ fn main(){
 To get all the files with a specific extension in a directory, use:
 
 ```rust
-use rust_search::Search;
+use rust_search::SearchBuilder;
 
-Search::new("/path/to/directory", None, Some(".fileExtension"), Some(1));
+let _ = SearchBuilder::default()
+    .location("/path/to/directory")
+    .ext("file_extension")
+    .depth(1)
+    .build();
```

Comment:
where is this `build` function definition... Should we name it `search()` ? That would make more sense probably

Reply:
Any builder should have `build` instance function, and it'll build